eks-cluster.tf

```diff
@@ -56,6 +56,24 @@ module "eks" {
       capacity_type  = local.config.capacity_type
       k8s_labels     = local.standard_tags
 
+      update_config = {
+        max_unavailable_percentage = local.config.max_unavailable_percentage # or set `max_unavailable`
+      }
+    }
+    apps-public = {
+      subnets                = module.vpc.public_subnets
+      public_ip              = true
+      create_launch_template = true
+      desired_capacity       = 1
+      max_capacity           = 3
+      min_capacity           = 1
+
+      instance_types = [local.config.node_instance_type]
```

Comment:
please create and use separate variable so that we can have different instance types for blockchain nodes than apps.

Reply:
@dincho i'm thinking should we have all configuration different for nodepools? not only node instance_type? capacity etc what you think?